apps/sharebymail/lib/ShareByMailProvider.php

```diff
@@ -354,8 +354,7 @@ protected function sendEmail(IShare $share, array $emails): void {
 		}
 
 		$emailTemplate->addBodyText(
-			htmlspecialchars($text . ' ' . $this->l->t('Click the button below to open it.')),
-			$text
+			$this->l->t('Click the button below to open it.')
```

Comment:
```suggestion
			htmlspecialchars($this->l->t('Click the button below to open it.'))
```
The translation may contain special characters depending on the language, so you should still escape them.

Reply:
Also I think `$text` is now unused and show be removed?

Reply:
> The translation may contain special characters depending on the language, so you should still escape them.

Hum...but all translations are unescaped, unless they have direct user input (like `$text` or `$note`)...are you absolutely sure here?

Reply:
> Also I think `$text` is now unused and show be removed?

Fixed 👍